src/templates/next/layouts/Homepage/Homepage.tsx

```diff
@@ -10,9 +11,13 @@ const HomepageLayout = ({
 }: HomePageSchema) => {
   return (
     <Skeleton site={site} page={page}>
-      {content.map((component) =>
-        renderComponent({ component, LinkComponent }),
-      )}
+      <div
+        className={`[&_.${ComponentContent}]:max-w-[1240px] [&_.${ComponentContent}]:px-6 [&_.${ComponentContent}]:md:px-10 [&_.${ComponentContent}]:mx-auto`}
```

Comment:
question: might be trippin' but [on Chromatic](https://65dd949d255950353c94d5cc-tqzldlvqhy.chromatic.com/iframe.html?args=&id=next-layouts-homepage--default&viewMode=story) it doesn't seem like the max width is applied?

<img width="1511" alt="Screenshot 2024-03-28 at 08 22 24" src="https://github.com/isomerpages/isomer-components/assets/27919917/f8b170ea-9621-4c9e-8eed-e6e1e99aec04">

Reply:
😭 works locally need to debug